### ISSUE TITLE
various profile page issues

### ISSUE THREAD

**Issue by Djfe** [mozilla/voice-web#1642]
Was that even intended?
It's buggy and not finished, yet.

lots of examples follow:
![firefox_2018-11-29_02-22-31](https://user-images.githubusercontent.com/5685512/49193268-d46ad480-f37e-11e8-98e5-61999602d655.png)
English/United States English is listed twice for some reason.

after that I visited the avatar page
Gravatar isn't working (EDIT: this is due to not being able to click on buttons on those new pages on desktop (I can connect to Gravatar on mobile))

after that I visited the goals page:
Error -> reload or report the error

after reloading the page it works

German and English (my two languages don't work yet though), only all languages (EDIT: this works fine on mobile, seems like the page doesn't register my clicks on these links and thats all)

I went back to profile:
united states is now shown only once (like it should be)

the checkbox whether I want mails to kept up to date is suddenly unchecked

I check it and save -> now it's gone and I can't uncheck it (EDIT: I just realized that this now moved from profile to settings)

![firefox_2018-11-29_02-35-36](https://user-images.githubusercontent.com/5685512/49193474-78ed1680-f37f-11e8-8e4a-c2262e453fd0.png)
I can't expand the text below "Why does this matter" anymore
EDIT: I also can't click on the emails on the settings page (privacy policy and terms). I can click on "skip feedback submission" though

On mobile clicking works just fine

I just noticed that my stats above changed:
![firefox_2018-11-29_02-37-19](https://user-images.githubusercontent.com/5685512/49193551-b782d100-f37f-11e8-8ea0-bfa11f7d6a5b.png)
one of the numbers was over 300 (I don't remember which one)
now it's: 261|275

I used common voice on several pcs and my smartphone in the past, maybe that's the issue and they weren't merged correctly? (all browsers were logged in with the same emailaddress and username)

**Comment by Djfe** [comment]
![firefox_2018-11-29_02-40-54](https://user-images.githubusercontent.com/5685512/49193670-36780980-f380-11e8-8fd6-640e8d33ff21.png)
Infinity (bug?)

not all strings are translated, yet.

**Comment by Djfe** [comment]
also: it's fun to see the statistics tool work while you watch a page lol
refresh and the numbers adjust

at first: 5 recordings (huh that can't be right)
switch to validations and back (ah now the numbers match up ^^)

**Comment by Djfe** [comment]
![firefox_2018-11-29_02-45-39](https://user-images.githubusercontent.com/5685512/49193817-e0579600-f380-11e8-8ba0-a936aa4e11f9.png)
is lower-case intentional?

**Comment by Djfe** [comment]
![firefox_2018-11-29_02-48-31](https://user-images.githubusercontent.com/5685512/49193924-578d2a00-f381-11e8-8466-51fb7facf67c.png)
If I click very fast repeatedly on the three dots button, then the last entry will be displayed as many times, as I pressed too much.
I can make mikehenrty appear 4 times at the end of that list (maybe more)

**Comment by Djfe** [comment]
the infinity seems to go away, when the statistics catch up with the db

**Comment by Djfe** [comment]
Before logging in on my smartphone:
![screenshot_20181129-025301](https://user-images.githubusercontent.com/5685512/49194228-8eb00b00-f382-11e8-8bd4-2f96dc2c0299.png)
After logging in on my smartphone:
![screenshot_20181129-025531](https://user-images.githubusercontent.com/5685512/49194231-91aafb80-f382-11e8-8475-8d52d08985b0.png)

**Comment by Djfe** [comment]
Managing subscriptions only links to the english version of that webpage instead of the localized one.

The icon on the edit button next to your emailaddress only shows up on mobile, but on desktop it's still there, just invisible (the button label is off-center)

**Comment by Djfe** [comment]
also: editing the mailaddress doesn't work.
The input is disabled and when I click on edit, I get to a login page.
After I login I get the error, that the (new) emailaddress is in use by another account already.

**Comment by Djfe** [comment]
![chrome_2018-11-29_03-15-15](https://user-images.githubusercontent.com/5685512/49194861-0717cb80-f385-11e8-9b60-137f6e0e15ea.png)
in chrome the opened/selected thing doesn't have an icon for some reason.
In Firefox it shows the black icon (instead of the grey one)

also: selecting one of them with tab you can see that it is so wide that it overlays the content div with it's border
![chrome_2018-11-29_03-19-33](https://user-images.githubusercontent.com/5685512/49195002-9de48800-f385-11e8-9efb-28d569ef3ea1.png)

**Comment by Djfe** [comment]
For some reason the input is not invisible
![screenshot_20181129-032417](https://user-images.githubusercontent.com/5685512/49195413-4b0bd000-f387-11e8-8ab8-28c7401c3afe.png)
Also: drag and drop can't work on mobile

**Comment by Djfe** [comment]
There is no profile button on mobile, I need to access dashboard or settings first to get there.

**Comment by Djfe** [comment]
seems like the div (class links) overlays the div (class content) preventing some of the links to be pressed/the text in div from being selected

**Comment by Djfe** [comment]
front page:
![chrome_2018-11-29_04-50-05](https://user-images.githubusercontent.com/5685512/49198584-181c0900-f394-11e8-8683-547dd0131c8d.png)
dashboard:
![chrome_2018-11-29_04-50-31](https://user-images.githubusercontent.com/5685512/49198586-1b16f980-f394-11e8-9cbd-34e674918ccd.png)
difference:
dashboard shows current time first (sorted the other way around), but marks the oldest time as the current time (green) (the bar at the end)

**Comment by Djfe** [comment]
"Profile" navigation element is displayed on top of the text in Chrome right now:
![chrome_2018-11-29_05-08-50](https://user-images.githubusercontent.com/5685512/49198736-ed7e8000-f394-11e8-88f3-cb03013a060b.png)
on hovering:
![chrome_2018-11-29_05-09-06](https://user-images.githubusercontent.com/5685512/49198738-ef484380-f394-11e8-92b9-677b9fcac5c9.png)

**Comment by Gregoor** [comment]
No accident, we soft-launched this (without major announcement), so that we could test it in the wild. Thanks for reporting those issues! I'll try to look into the critical ones today.

**Comment by Djfe** [comment]
I'll help later on in ordering my gibberish and answering my questions ^^
I was mostly was on a spree to hunt bugs, so I collected them first. I also tried to be fast with it to show how much is affected (since I thought the launch was accidental)

I'll also add some videos (for example to showcase how the email edit bug looks to me)

**Comment by Gregoor** [comment]
It's much appreciated! Thanks for helping beta test it and making the
experience solid.

**Comment by rprys** [comment]
Issue: I couldn't log in - the full login page with choices to log in via Firefox or Google was gone. 
The page asked for my e-mail and LDAP, which it rejected... ;-(

**Comment by Djfe** [comment]
I can confirm that.
only login via email and I don't want to open a new account, so I aborted once it wanted to send a verification email.

**Comment by mbransn** [comment]
@rprys @Djfe apologies for the confusion re: login options. All login option other than email weren’t meant to be available for Common Voice quite yet. As they require 2FA and this can cause errors / difficulty for some contributors, email verification is the login method we’d prefer to support for now as it is the most inclusive. 

All accounts are identified by email address, so if you use the same address you use on Github or Firefox Accounts, you should be able to log back into your profile. If that is not the case, please file a separate issue and let us know. Thanks for your patience as we work through this launch! 

cc @hmitsch
